.github/workflows/test.yml

```diff
@@ -31,3 +31,4 @@ jobs:
           # For now, you'll need to manually specify a GitHub token until we
           # solve #2. The x: prefix is a dummy username.
           token: x:${{ secrets.GITHUB_TOKEN }}
+          dry-run: true
```

Comment:
Perhaps we can make this `true` for PRs, and `false` for commits to `main`? That way it continues to be a true dogfood test when possible.